Source/diablo.cpp

```diff
@@ -498,6 +500,7 @@ void diablo_quit(int exitStatus)
 int DiabloMain(int argc, char **argv)
 {
 	diablo_parse_flags(argc, argv);
+	LoadOptionsFromConfig(); //Fluffy: Read options from config here

```

Comment:
look how your function name stands out from the surrounding ones

Reply:
It doesn't stand out compared to "DiabloMain()"

Do we have standards for how functions should be named in Devilution?

Reply:
CamelCase is the standard. The rest is names that should be cleaned up.